docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,625 @@
+
+#### Create Provider
+* Create Azure provider:
+```bash
+sed "s/BASE64ENCODED_AZURE_PROVIDER_CREDS/$BASE64ENCODED_AZURE_PROVIDER_CREDS/g" azure-provider.yaml | kubectl create -f -
+```
+
+* Verify Azure provider was successfully registered by the crossplane
+```bash
+kubectl get providers.azure.crossplane.io -n azure-infra-dev
+kubectl get secrets -n azure-infra-dev
+```
+
+#### Cloud-Specific Resource Classes
+Cloud-specific resource classes are used to define a reusable configuration for
+a specific managed service. Wordpress requires a MySQL database, which can be
+satisfied by an [Azure Database for
+MySQL](https://azure.microsoft.com/en-us/services/mysql/) instance.
+
+* Define an Azure MySQL `SQLServerClass` in `azure-mysql-standard.yaml`:
+```yaml
+cat > azure-mysql-standard.yaml <<EOF
+---
+apiVersion: database.azure.crossplane.io/v1alpha2
+kind: SQLServerClass
+metadata:
+  name: azure-mysql-standard
+  namespace: azure-infra-dev
+specTemplate:
+  adminLoginName: myadmin
+  resourceGroupName: RESOURCE_GROUP_NAME
+  location: EAST US
+  sslEnforced: false
+  version: "5.6"
+  pricingTier:
+    tier: GeneralPurpose
+    vcores: 2
+    family: Gen5
+  storageProfile:
+    storageGB: 25
+    backupRetentionDays: 7
+    geoRedundantBackup: false
+  providerRef:
+    name: demo-azure
+    namespace: azure-infra-dev
+  reclaimPolicy: Delete
+EOF
+```
+
+* Create the `SQLServerClass`:
+```bash
+sed "s/RESOURCE_GROUP_NAME/$RESOURCE_GROUP_NAME/g" azure-mysql-standard.yaml | kubectl apply -f -
+```
+
+* You should see the following output:
+```bash
+sqlserverclass.database.azure.crossplane.io/azure-mysql-standard created
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get sqlserverclasses -n azure-infra-dev
+```
+*Output*
+```bash
+NAME                   PROVIDER-REF   RECLAIM-POLICY   AGE
+azure-mysql-standard   demo-azure     Delete           11s
+```
+
+You are free to create more Azure `SQLServerClass` instances to define more
+potential configurations. For instance, you may create `large-azure-mysql` with
+field `storageGB: 100`.
+
+#### Application Namespaces
+Earlier, we created a namespace to group our Azure infrastructure resources.
+Because our application resources may be satisfied by services from any cloud
+provider, we want to separate them into their own namespace. For this demo, we
+will create a namespace called `app-project1-dev`, which we will use to group
+our Wordpress resources.
+
+* Define a `Namespace` in `app-project1-dev-namespace.yaml`:
+```yaml
+cat > app-project1-dev-namespace.yaml <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: app-project1-dev
+EOF
+```
+
+* Create the `Namespace`:
+```bash
+kubectl create -f app-project1-dev-namespace.yaml
+```
+
+* You should see the following output:
+```bash
+namespace/app-project1-dev created
+```
+
+#### Portable Resource Classes
+Portable resource classes are used to define a class of service in a single
```

Comment:
What is "portable" in this context? What is a "class of service" in this context? I would suggest that we use more accessible terminology, or define these concepts before we use them, or link to documents which define the concepts. Or maybe all three! Haha

Reply:
Related to another comment below, I think it is reasonable to:

1. Assume that this is not the first part of the docs that a new user would look at.
1. Assume that if this is the part doc they come to, it is intuitive to look at an earlier part of the docs if they desire more in-depth explanation of a concept.

I worry that re-explaining every concept in every doc will make every guide very dense. However, I definitely agree that linking would be good. I am currently somewhat hampered in this effort because I am not always sure where the relevant doc is going to end up.